packages/feedback-service/src/feedback-config/schema.ts

```diff
@@ -4,9 +4,15 @@ import {
 
 export const FeedbackConfigSchema: Schema = new Schema({
   appId: {
+    type: String,
+    required: false,
+    unique: false,
+  },
+  projectId: {
     type: String,
     required: true,
     unique: true,
+    default: 'home-spa',
```

Comment:
I thnk defaulting the same value can make conflicts

Reply:
We can remove the default

Reply:
Done